src/components/Card/Card.tsx

```diff
@@ -7,25 +7,48 @@ import {
   useStyleConfig,
 } from "@chakra-ui/react";
 
-import { CardLayouts } from "./CardTypes";
+import { LayoutTypes } from "../../helpers/enums";
 import Heading from "../Heading/Heading";
 import Image, { ImageProps } from "../Image/Image";
 import { ImageRatios, ImageSizes } from "../Image/ImageTypes";
 import generateUUID from "../../helpers/generateUUID";
 
 interface CardBaseProps {
   /** Optional value to control the alignment of the text and elements. */
-  center?: boolean;
+  isCentered?: boolean;
   /** Optional value to render the layout in a row or column.
-   * Default is `CardLayouts.Column`. */
-  layout?: CardLayouts;
+   * Default is `LayoutTypes.Column`. */
+  layout?: LayoutTypes;
 }
 
 interface CardLinkBoxProps {
   /** Main link to use when the full `Card` component should be clickable. */
   mainActionLink?: string;
 }
 
+// Used internally only for the `imageProps` prop for the `Card` component.
```

Comment:
Kind of. I'm pulling in `ImageProps` for the `CardImage` component. Maybe the naming could be slightly better.

The other place something like this happens in the `StructuredContent` but the only difference is this has `isAtEnd` and `StructuredContent` has `position`. Otherwise, I can update this.